packages/integration_test/lib/channel.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+
+/// The method channel used to report the result of the tests to the platform.
+/// On Android, this is relevant when running instrumented tests.
+const MethodChannel integrationTestChannel = MethodChannel('plugins.flutter.io/integration_test');
```

Comment:
I'm pretty sure we don't need this exposed publicly in the package right? Can we just move this to `src` so that developers do not end up depending on it?

Reply:
done